web/wp-content/themes/dctx/single.php

```diff
@@ -5,9 +5,21 @@
 
 		<div class="main-content">
 			<?php while (have_posts()) : the_post(); ?>
```

Comment:
:no_entry_sign: **Error**: No space after opening parenthesis is prohibited (*WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceAfterOpenParenthesis*).